sdk/formrecognizer/azure-ai-formrecognizer/migration-guide.md

```diff
@@ -0,0 +1,597 @@
+and `DocumentModelAdministrationAsyncClient`, instantiated via the [DocumentModelAdministrationClientBuilder][DocumentModelAdministrationClientBuilder].
+The sync and async operations are separated to [DocumentModelAdministrationClient][DocumentModelAdministrationClient] and [DocumentModelAdministrationAsyncClient][DocumentModelAdministrationAsyncClient].
+
+Instantiating FormRecognizerClient client with 3.x.x:
+```java
+FormTrainingClient formTrainingClient = new FormTrainingClientBuilder()
+    .credential(new AzureKeyCredential("{key}"))
+    .endpoint("{endpoint}")
+    .buildClient();
+```
+
+Instantiating DocumentModelAdministrationClient client with 4.x.x:
+<!-- embedme ./src/samples/java/com/azure/ai/formrecognizer/ReadmeSamples.java#L62-L65 -->
+```java
+DocumentModelAdministrationClient documentModelAdminClient = new DocumentModelAdministrationClientBuilder()
+    .credential(new AzureKeyCredential("{key}"))
+    .endpoint("{endpoint}")
+    .buildClient();
+```
+
+#### Analyze documents
+Analyze data from certain types of common documents (such as receipts, invoices, business cards, or identity documents) 
+using prebuilt models or from custom documents using your own custom models and extract table and structured layout data.
+
+With 4.x.x, the unified method, `beginAnalyzeDocument` and `beginAnalyzeDocumentFromUrl`:
+- accepts a string type `modelId` to be any of the prebuilt model IDs or a custom model ID.
+- the return type `AnalyzeResult` model now exposes document elements, such as key-value pairs, entities, tables,
+  document fields and values at the top level of the returned model. As compared to the previously returned model
+  `RecognizedForm` which included hierarchical relationships between `FormElements` for instance tables were an element
+  of a `FormPage` and not a top-level element.
+- provides the functionality of `beginRecognizeCustomForms`, `beginRecognizeContent`, `beginRecognizeReceipt`,
+  `beginRecognizeReceipts`, `beginRecognizeInvoices` `beginRecognizeIdentityDocuments` and `beginRecognizeBusinessCards` from the previous (azure-ai-formrecognizer 3.1.X - below) package versions.
+- accepts unified `AnalyzeDocumentOptions` to specify pages and locale information for the outgoing request
+- the `includeFieldElements` parameter is not supported with the `DocumentAnalysisClient`, text details are automatically included with API version `2021-09-30-preview` and later.
+- the `readingOrder` parameter does not exist as the service uses `natural` reading order for the returned data.
+
+#### Using a prebuilt model
+- In 3.x.x, `beginRecognizeReceipts` and `beginRecognizeReceiptsFromUrl` method was used to analyze receipts.
+- In 4.x.x, `beginRecognizeReceipts` and `beginRecognizeReceiptsFromUrl` has been replaced with `beginAnalyzeDocument` and `beginAnalyzeDocumentFromUrl` respectively.
```

Comment:
mention that this applies to all prebuilts, not just receipts?

Reply:
Yeah I agree it's better to make that clearer